File: SKKU/algorithms/quad_for_iter.c
#include <stdio.h>

void main() {
    int sum = 0;
    int n = 10;
    for (int i = 0; i < n; i++)
        for (int j = 0; j < i; j++) 
            for (int k = 0; k < j; k++) 
                sum++;
    printf("Total complexiity: %d\n", sum);
}

File: SKKU/algorithms/quad_for_iter.rs


fn main() {
    let mut sum:isize = 0;
    let n:isize = 10;
    for i in 0..n {
        for j in 0..i { 
            for _k in 0..j {
                sum += 1;
            }
        }
    }

    println!("Total complexiity: {}", sum);
}

